### ISSUE TITLE
Code release consulting

### ISSUE THREAD

**Issue by Sarlren** [Drexubery/ViewCrafter#6]
Hi, this work is awesome. I notice some part of the code contains unreleased part like sparse-view synthesis. I would like to consult for the release time of the rest part. Thank you!

**Comment by hiyyg** [comment]
+1

**Comment by Drexubery** [comment]
Thanks! We are still working hard on cleaning the code, please stay tuned.

**Comment by Vincento-Wang** [comment]
> Thanks! We are still working hard on cleaning the code, please stay tuned.

will the code of all training part be released ?

**Comment by Drexubery** [comment]
> @Drexubery will the code of all training part be released ? I read many papers, I think your work is really the best baseline and codebase for future work.

Hi, thanks for your comments!
We are currently working on a new project to train a dynamic video novel view synthesis model. We will release the complete training code once the project is complete.
